packages/react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -58,7 +60,7 @@ const Checkbox = React.forwardRef<CheckboxElement, CheckboxProps>((props, forwar
   });
```

Comment:
It may be beneficial to add a couple assertions to check that these classnames are present since we support this way of styling now. This will help with backwards compatibility.

Reply:
I'm trying to avoid classes or style testing since that is really brittle. Our visual  regression tests should cover those